src/main/java/io/pravega/connectors/flink/sink/PravegaEventWriter.java

```diff
@@ -228,8 +228,8 @@ public void close() throws Exception {
     }
 
     @VisibleForTesting
```

Comment:
Here we mark it `VisibleForTesting`, but I'm not seeing it being used in tests. The coverage is also below par, needs to add some more test to cover